script/prepare-llvm-linux.sh

```diff
@@ -48,7 +48,8 @@ $CP -r llvm/include/*-*-* llvm-host/include/
 $CP $GLIBC/lib/libc_nonshared.a stage1/lib/glibc
 for f in $GLIBC/lib/lib{c,dl,m,rt,pthread}-*; do b=$(basename $f); cp $f stage1/lib/glibc/${b%-*}.so; done
 OPTIONS=()
-echo -n " -DLLVM=ON -DLLVM_CONFIG=$PWD/llvm-host/bin/llvm-config" # manually point to `llvm-config` location
```

Comment:
@Kha do we not need to point our build to the correct `llvm-config` which we download from [`leanprover/lean-llvm`](https://github.com/leanprover/lean-llvm)?

Reply:
Oh yes, Linux LLVM should use this file as well and this line should become conditional. I can do the change of desired.

Reply:
Yes please, that would be very super helpful!

Reply:
Yes, this seems to now all go through, modulo the one remaining change to make this line conditional. @Kha, if you could take care of this that would be much appreciated.

Reply:
Thanks @hargoniX!